.cirrus.yml

```diff
@@ -77,6 +77,7 @@ task:
         cpu: 4
         memory: 12G
     - name: web_tests-linux
+      allow_failures: true
```

Comment:
Maybe add a TODO to remove this ones these are stable.

Reply:
Done